### PR TITLE
Pipeline S3 config update

### PR DESCRIPTION
**Pull request by jsitu777**
**Description of your changes:**
- Modify s3 config to update artifacts keyformat with date to avoid name collisions 
- use emissary as containerRuntimeExecutor for workflow-controller-configmap
- update associated pipeline chart content changes with this PR https://github.com/awslabs/kubeflow-manifests/pull/654

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsconfigs/apps/pipeline-static/s3/config [hunk 1]
```diff
@@ -6,15 +6,10 @@ artifactRepository:
         keyPrefix: artifacts,
         endpoint: $(kfp-artifact-storage-endpoint),
         insecure: false,
-        accessKeySecret: {
```
Comment (ryansteakley) [inline awsconfigs/apps/pipeline-static/s3/config:9]: why is this part removed from pipeline-static ? These are used for static-credentials, has something changed?

Comment (jsitu777) [reply awsconfigs/apps/pipeline-static/s3/config:9]: it was a mistake, just fixed.

**Review by surajkota — APPROVED** [review]

**Comment by surajkota** [comment]
@ryansteakley please review it from your end

**Review by ryansteakley — APPROVED** [review]
